app/controllers/album.js

```diff
@@ -0,0 +1,26 @@
+const User = require('../models').users,
+  logger = require('../logger'),
+  https = require('https'),
+  config = require('../../config');
+
+exports.list = (req, res, next) => {
+
+  logger.info(`Attempting GET request to url ${config.common.urlRequests.albumList}`);
+
+  https.get(config.common.urlRequests.albumList, (resp) => {
```

Comment:
This solution works fine, but it has the great disadvantage that the response its a stream, which may be hard to handle. That's why we usually prefer to use another package that has already promise based functions and allows us to make the same requests but handling them as the rest of the application, via Promises.
We usually use [request-promise](https://github.com/request/request-promise) as our http request package, but there are also some other interesant like axios.

Reply:
Also we like to divided responsibilities between the different code entities and for example when we have to make a request to an external service, we use to delegate that to a service and not to make it here in the controller.
If you want to read more about that we have the following [file](https://docs.google.com/document/d/1ufn2NBFJkGyDmfDQCSWjBHrAQku0ZfZemc8aDV97qk8/edit?usp=sharing) that explains a little bit one way of structuring an app.
Anyways that file is a little bit raw, so any suggestion or change can and will be made.